mlrun/api/runtime_handlers/daskjob.py

```diff
@@ -307,6 +307,9 @@ def enrich_dask_cluster(
         or "daskdev/dask:latest"
     )
     env = spec.env
+    env.extend(
+        [{"name": k, "value": v} for k, v in function.generate_runtime_env().items()]
+    )
```

Comment:
code dup with `mlrun/runtimes/mpijob/v1alpha1.py` 
```python
extra_env = self.generate_runtime_env(runobj)
extra_env = [{"name": k, "value": v} for k, v in extra_env.items()]
```

Perhaps you can have expose "generate_runtime_k8s_env" which uses `_ generate_runtime_env`. wdyt?

Reply:
Good thinking. I did what you suggested. See if it's looks better now.